#DataAcquired

# Time Acquisition

time time_now



# Current Joint position of the right hand

sensor_msgs/JointState right_hand_joint_curr_state



# Hand to vito anchor trasform

# index
geometry_msgs/TransformStamped right_hand_index_distal_link 
geometry_msgs/TransformStamped right_hand_index_knuckle_link
geometry_msgs/TransformStamped right_hand_index_middle_link
geometry_msgs/TransformStamped right_hand_index_proximal_link

#little
geometry_msgs/TransformStamped right_hand_little_distal_link
geometry_msgs/TransformStamped right_hand_little_knuckle_link
geometry_msgs/TransformStamped right_hand_little_middle_link
geometry_msgs/TransformStamped right_hand_little_proximal_link

#middle
geometry_msgs/TransformStamped right_hand_middle_distal_link
geometry_msgs/TransformStamped right_hand_middle_knuckle_link
geometry_msgs/TransformStamped right_hand_middle_middle_link
geometry_msgs/TransformStamped right_hand_middle_proximal_link

#ring
geometry_msgs/TransformStamped right_hand_ring_distal_link
geometry_msgs/TransformStamped right_hand_ring_knuckle_link
geometry_msgs/TransformStamped right_hand_ring_middle_link
geometry_msgs/TransformStamped right_hand_ring_proximal_link

#thumb
geometry_msgs/TransformStamped right_hand_thumb_distal_link
geometry_msgs/TransformStamped right_hand_thumb_knuckle_link
geometry_msgs/TransformStamped right_hand_thumb_proximal_link




#  Force / Torque Sensor

# ContactsState of the hand  in simulation ????? 
# index
gazebo_msgs/ContactsState right_hand_index_distal_cs
gazebo_msgs/ContactsState right_hand_index_knuckle_cs
gazebo_msgs/ContactsState right_hand_index_middle_cs
gazebo_msgs/ContactsState right_hand_index_proximal_cs

#little
gazebo_msgs/ContactsState right_hand_little_distal_cs
gazebo_msgs/ContactsState right_hand_little_knuckle_cs
gazebo_msgs/ContactsState right_hand_little_middle_cs
gazebo_msgs/ContactsState right_hand_little_proximal_cs

#middle
gazebo_msgs/ContactsState right_hand_middle_distal_cs
gazebo_msgs/ContactsState right_hand_middle_knuckle_cs
gazebo_msgs/ContactsState right_hand_middle_middle_cs
gazebo_msgs/ContactsState right_hand_middle_proximal_cs

#ring
gazebo_msgs/ContactsState right_hand_ring_distal_cs
gazebo_msgs/ContactsState right_hand_ring_knuckle_cs
gazebo_msgs/ContactsState right_hand_ring_middle_cs
gazebo_msgs/ContactsState right_hand_ring_proximal_cs

#thumb
gazebo_msgs/ContactsState right_hand_thumb_distal_cs
gazebo_msgs/ContactsState right_hand_thumb_knuckle_cs
gazebo_msgs/ContactsState right_hand_thumb_proximal_cs





# ContactsState of the arm 

gazebo_msgs/ContactsState right_arm_cs

gazebo_msgs/WorldState current_state	




# Current Motor position [degree] of the hand

float32 right_hand_synergy_joint_state_position
float32 right_hand_synergy_joint_state_velocity
float32 right_hand_synergy_joint_state_effort

# Current Motor Current  [A] of the hand

float32 motor_current


